test/linear_combination.jl

```diff
@@ -218,5 +226,13 @@
             @test Transforms.apply(df, lc; cols=cols) == expected
             @test lc(df; cols=cols) == expected
         end
+
+        @testset "single col" begin
+            lc_single = LinearCombination([-1])
+
+            @test Transforms.apply(df, lc_single; cols=:a) == [-1, -2, -3]
```

Comment:
Does this actually work for the string `cols="a"`? We are checking if the type is AbstractString but don't have tests for it

Reply:
I've refactored so it doesn't check `AbstractString` now, but the reason I did that is DataFrames supports indexing columns as strings.